src/resources/ProductListingMetrics/ProductListingMetricsInterfaces.ts

```diff
@@ -0,0 +1,23 @@
+export type ProductListingMetricsOptions = {
+    page?: number;
+    perPage?: number;
+    startDate?: string;
+    endDate?: string;
+    sort?: string;
+    filter?: string;
+};
+
+export interface ProductListingMetricsItem {
+    id: string;
+    name: string;
+    revenue: number;
+    lastNumberOfProducts: number;
+    catalogId: string;
+    conversionRate: number;
+    bounceRate: number;
+}
+export interface ProductListingMetricsModel {
+    items: ProductListingMetricsItem[];
+    totalPages: number;
+    totalEntries: number;
+}
```

Comment:
This interface isn't necessary, we can use `PageModel<ProductListingMetricsItem>` instead